lib/internal/modules/run_main.js

```diff
@@ -23,6 +27,10 @@ function resolveMainPath(main) {
   return mainPath;
 }
 
+/**
+ * Determine whether the main entry point should be loaded through the ESM Loader.
+ * @param {string} mainPath Absolute path to the main entry point
```

Comment:
returns boolean is missing

Reply:
Inferred.

<img width="675" alt="image" src="https://github.com/nodejs/node/assets/456802/0a9621c4-87a8-4acd-bc61-ce4ee39fa5a0">